core/src/main/java/com/graphhopper/storage/index/LocationIndexTree.java

```diff
@@ -564,6 +572,133 @@ protected boolean check(int node, double normedDist, int wayIndex, EdgeIteratorS
         return closestMatch;
     }
 
+    /**
+     * Returns all edges that are within the specified radius around the queried position.
+     *
+     * @param queryLat latitude to search from
+     * @param queryLon longitude to search from
+     * @param edgeFilter only include edges from edgeFilter
+     * @param radius include all results within this radius (in meters) - assuming it's not
+     *        bounded by maxIterations.
+     * @param maxIterations if not null, search only a square of (2 * maxIterations - 1) tiles
```

Comment:
> The purpose is to find everything within radius. The maxIterations is there to bound to a certain amounts of tiles - as discussed in MapMatching 65.

Sorry, I still don't really get it. Shouldn't this be the default? If we actually want something like this, could we create a boolean like: "includeSurroundingTiles". This would add increase the requiredRadius by one. I think this parameter requires too much knowledge about the actual method. Since this is a public method and is intended to be used by everyone, we should keep it simple.